metadata.json

```diff
@@ -15,7 +15,7 @@
     },
     {
       "operatingsystem": "RedHat",
-      "operatingsystemrelease": ["5","6","7","26", "27"]
+	"operatingsystemrelease": ["5","6","7","8","26", "27"]
```

Comment:
this indentation is two spaces too long

Reply:
fixed. The 26,27 is dubious. Those are fedora version numbers. And not even ones
your really supporting.